templates/dart/lib/src/chunked_upload_io.dart.twig

```diff
@@ -0,0 +1,49 @@
+import 'client.dart';
+import 'dart:io' as io;
+import 'enums.dart';
+import 'package:http/http.dart' as http;
+import 'response.dart';
+import 'dart:math';
+import 'input_file.dart';
+import 'exception.dart';
+
+Future<Response> chunkedUpload({
+  required Client client,
+  required String path,
+  required Map<String, dynamic> params,
+  required String paramName,
+  required Map<String, String> headers,
+}) async {
+  InputFile file = params[paramName];
+  if (file.path == null) {
+    throw AppwriteException("File path must be provided for dart:io");
+  }
+  io.File iofile = io.File(file.path!);
+  final size = await iofile.length();
+
+  late Response res;
+  if (size <= Client.CHUNK_SIZE) {
+    params['file'] = await http.MultipartFile.fromPath(paramName, file.path!, filename: file.fileName);
+    res = await client.call(HttpMethod.post,
+        path: path, params: params, headers: headers);
```

Comment:
In service.dart.twig (line 70 - 76) you place each argument on a different line. I like this makes readability super clear. Here, perhaps we could follow the same pattern?